quint/io-cli-tests.md

```diff
@@ -625,3 +625,33 @@ exit $exit_code
 ```
 error: --seed must be a big integer, found: NotANumber
 ```
+
+### OK on compile imports
+
+<!-- !test in compile imports -->
+```
+echo "init" | quint -r ../examples/language-features/imports.qnt::G 2>&1 | tail -n +3
+```
+
+<!-- !test out compile imports -->
+```
+true
+
+>>> true
+>>> 
+```
+
+### OK on compile instances
+
+<!-- !test in compile instances -->
+```
+echo "A2::g(false)" | quint -r ../examples/language-features/instances.qnt::Instances 2>&1 | tail -n +3
```

Comment:
`g` has the same behavior in `A1` and `A2`. Would it make sense to test `f` too, which has different behaviors in `A1` and `A2`?